app/assets/stylesheets/application/layout.scss

```diff
@@ -26,8 +26,15 @@
     }
   }
   .update {
```

Comment:
What kinds of things could be in the update container besides `<p>`, that would need smaller font-size and line-height ratio?

Reply:
I'll ask Soraya.

Reply:
She says it'll just be text except in the future she'd like a date field (I've added an issue for that).